processors/filterprocessor/gen/generate.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"go/format"
+	"os"
+	"reflect"
+	"sort"
+	"strings"
+
+	"github.com/algorand/go-algorand/data/transactions"
```

Comment:
Does this work if you use the transaction object from the go SDK?

Reply:
Perhaps.  Is it easy to test by importing the sdk?

Reply:
Indexer uses the SDK, so I think you could swap in the other type: https://github.com/algorand/go-algorand-sdk/blob/develop/types/transaction.go#L25

It looks like `SignedTxnWithAD` isn't available, but I'm not sure that one is especially useful so maybe this is OK?

Reply:
Good point about the IDs, that might be the only place it is available when things are being created.